tests/acceptance/21_methods/reachable/defined_bundle_usebundle_parameters.cf

```diff
@@ -0,0 +1,19 @@
+# These tests are left very simple on purpose,
+# they are not using the test framework.
+# They should be seen together, note that defined vs undefined variants are
+# almost identical, just replacing one detail to show what should work
+# and what should error.
+
+bundle agent bar(x)
+{
+  reports:
+    "$(x)";
+    "$(this.promise_filename) Pass";
+}
+
+bundle agent main
+{
+  methods:
+    "foo"
+      usebundle => bar("baz");
```

Comment:
I don't understand this corner case. Maybe a comment would help ;)

Reply:
I could add a comment but it would say the same as it says in the top of this file. This test doesn't make sense on it's own, but when you look at it together with the other test with undefined in the name it makes a bit more sense. The one where it's defined should pass, and the one where it's undefined should fail. They are identical except changing that one bundle name.